### DISCUSSION TITLE
Filter view in Android TV app

### DISCUSSION THREAD

**Discussion by tursoe** [jellyfin/jellyfin-androidtv#4300] [General]
Hi. I'm almost new to Jellyfin, for many years I have used Plex but see more and more problems. For the last year I installed Jellyfin in a Docker container on my server and used a lot of time adding my media, edit metadata and create the right experience for my kids with age restrictions.
My media library is great on the web and my Android phone I'll used for testing and configuration the last year so now before Christmas I wanted to install Jellyfin on our TVs and have the time around them in the beginning if anything isn't right.
First thing I noticed is you can't search by genres, age restrictions, not watched, years and so on in the Android TV app.

Are there any plan to support this in the future? Maybe I'm going to use Kodi with Jellyfin plugin for now so I can turn off the old not maintained Plex server.

Best regards

**Comment by voc0der** [comment]
Hello, bumping this! Would love to see this feature.

**Comment by pedroarruda29** [comment]
Same here. 
When I'm using the Android TV with my kids, it would be great to be able to filter by genre (e.g. Animation). 
Right now, I open the app on my phone, filter for animation, choose the movie I want to watch, and only after I search for the movie on the Android TV.
Would be great to be able to filter that directly on the Android TV app.
